plugin/evm/vm.go

```diff
@@ -740,28 +728,28 @@ func (vm *VM) onNormalOperationsStarted() error {
 	ctx, cancel := context.WithCancel(context.TODO())
 	vm.cancel = cancel
 
-	// update validators first
-	if err := vm.performValidatorUpdate(ctx); err != nil {
+	// sync validators first
+	if err := vm.validatorsManager.Sync(ctx); err != nil {
```

Comment:
Should we move the initialization logic (from here to line 747) to the validatorsManager?
such as:
```suggestion
	if err := vm.validatorsManager.Init(ctx); err != nil {
```

One small complication seems the shutdownWg, which we could share or add a .Stop or .Shutdown method to the manager with its own wg

Reply:
I think in general VM might want to manage processes directly, at least this is the current approach.

Reply:
IMO we should steer the VM to being more modular with a bunch of composable/optional components.

However this PR is already a big improvement so I am going to approve it and we can deal with this later.